src/app.js

```diff
@@ -11,14 +11,16 @@ const testsRouter = require("./routes/testsRouter");
 const activitiesRouter = require("./routes/activitiesRouter");
 const authRouter = require("./routes/authRouter");
 const votesRouter = require("./routes/votesRouter");
+const groupsRouter = require("./routes/groupsRouter");
 
 // api documentation: swagger-ui
-const swaggerDocument = require("yamljs").load("./src/swagger.yaml");
+const swaggerDocument = require("yamljs").load("./swagger.yaml");
```

Comment:
I'm not sure why I can't get this to work - for some reason when I change it back to the '/src' path, I get the following error: 

Error: ENOENT: no such file or directory, open './src/swagger.yaml'

I'll try puttering around a bit more, but might move this discussion to Slack in case anyone has any ideas on how to resolve.

Reply:
Yes, this change works for me.

Reply:
Works for me too!